frontend/src/component/common/AvatarGroup/AvatarGroup.tsx

```diff
@@ -73,16 +76,22 @@ const AvatarGroupInner = ({
         [users],
     );
 
+    const overflow = users.length - avatarLimit;
+
     return (
         <StyledAvatars className={className}>
             {shownUsers.map((user) => (
                 <AvatarComponent key={objectId(user)} user={user} />
             ))}
             <ConditionallyRender
-                condition={users.length > avatarLimit}
+                condition={overflow > 0}
                 show={
-                    <AvatarComponent>
-                        +{users.length - shownUsers.length}
+                    <AvatarComponent
+                        user={{
+                            username: `+${millify(overflow)}`,
+                        }}
+                    >
+                        +{Math.min(overflow, MAX_OVERFLOW_DISPLAY_NUMBER)}
```

Comment:
Personally I was confused by the + sign here. Maybe sth like: `Total: 3452` would be more obvious?
But no strong opinion here so I'm also ok with the current approach.

Reply:
Do you mean in the tooltip or in the avatar circle? I was considering doing that for the tooltip, yeah. So if that's what you mean, I'll do that before merging. If it's in the circle, then I don'th think we have that much space 😅

Reply:
tooltip

Reply:
yeah, cool. then we're aligned. I'll update